custom/icds_reports/utils/aggregation_helpers/agg_awc.py

```diff
@@ -0,0 +1,512 @@
+                sum(wer_eligible) AS wer_eligible
+            FROM agg_child_health
+            WHERE month = %(start_date)s AND aggregation_level = 5 GROUP BY awc_id, month
+        ) ut
+        WHERE ut.month = agg_awc.month AND ut.awc_id = agg_awc.awc_id;
+        """.format(
+            tablename=self.tablename,
+        ), {
+            'start_date': self.month_start
+        }
+
+        yield """
+        UPDATE "{tablename}" agg_awc SET
+            cases_ccs_pregnant = ut.cases_ccs_pregnant,
+            cases_ccs_lactating = ut.cases_ccs_lactating,
+            cases_ccs_pregnant_all = ut.cases_ccs_pregnant_all,
+            cases_ccs_lactating_all = ut.cases_ccs_lactating_all,
+            cases_person_beneficiary_v2 = COALESCE(cases_person_beneficiary_v2, 0) + ut.cases_ccs_pregnant + ut.cases_ccs_lactating
+        FROM (
+            SELECT
+                awc_id,
+                month,
+                sum(pregnant) AS cases_ccs_pregnant,
+                sum(lactating) AS cases_ccs_lactating,
+                sum(pregnant_all) AS cases_ccs_pregnant_all,
+                sum(lactating_all) AS cases_ccs_lactating_all
+            FROM agg_ccs_record
+            WHERE month = %(start_date)s AND aggregation_level = 5 GROUP BY awc_id, month
+        ) ut
+        WHERE ut.month = agg_awc.month AND ut.awc_id = agg_awc.awc_id;
+        """.format(
+            tablename=self.tablename,
+        ), {
+            'start_date': self.month_start
+        }
+
+        yield """
+        UPDATE "{tablename}" agg_awc SET
+           cases_household = ut.cases_household
+        FROM (
+            SELECT
+                owner_id,
+                sum(open_count) AS cases_household
+            FROM "{household_cases}"
+            GROUP BY owner_id
+       ) ut
+        WHERE ut.owner_id = agg_awc.awc_id
+        """.format(
+            tablename=self.tablename,
+            household_cases=self._ucr_tablename('static-household_cases'),
+        ), {}
+
+        yield """
+        UPDATE "{tablename}" agg_awc SET
+           cases_person = ut.cases_person,
+           cases_person_all = ut.cases_person_all,
+           cases_person_adolescent_girls_11_14 = ut.cases_person_adolescent_girls_11_14,
+           cases_person_adolescent_girls_11_14_all = ut.cases_person_adolescent_girls_11_14_all,
+           cases_person_adolescent_girls_15_18 = ut.cases_person_adolescent_girls_15_18,
+           cases_person_adolescent_girls_15_18_all = ut.cases_person_adolescent_girls_15_18_all,
+           cases_person_referred = ut.cases_person_referred
+        FROM (
+            SELECT
+                awc_id,
+                sum(seeking_services) AS cases_person,
+                sum(count) AS cases_person_all,
+                sum(CASE WHEN %(month_end_11yr)s > dob AND %(month_start_15yr)s <= 'dob' AND sex = 'F' THEN seeking_services ELSE 0 END) as cases_person_adolescent_girls_11_14,
```

Comment:
I think you need to remove the quotes from dob in this line and the following ones